app/models/infrastructure.rb

```diff
@@ -143,4 +143,10 @@ def component_hash(type, count)
     name = "#{self.cluster_name}-#{type}-#{format('%02d', count.to_i)}"
     { name: name, type: type}
   end
+
+  def self.options_for_select
+    infrastructures = Infrastructure.arel_table
+    # order('LOWER(name)').map { |e| [e.name, e.id] }
```

Comment:
Please remove this unnecessary comment

Reply:
Done https://github.com/BaritoLog/BaritoMarket/pull/134/commits/415056c98c6b193a5d1a3f996e160d268406725e